fragment.fs

```diff
@@ -0,0 +1,16 @@
+uniform sampler2D u_Texture;
```

Comment:
How do these files work? Is the GPU filling the globals with each fragment data? There are a lot more questions that come with that but these are the bare minimum to understand a bit more how it runs.

Reply:
Understanding the syntax more might help us generate our own